Sources/CardPayments/Models/Address.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 /// The postal address.
-public struct Address: Codable {
+public struct Address: Codable, Equatable {
```

Comment:
q: curious is `Equatable` conformance needed here?

Reply:
I do comparison of the Addresses in a unit test and I didn't want
to have to compare each components of the address.

Reply:
I think you may be able to declare an extension like

```swift
extension Address: Equatable {}
```
in the unit test if needed. May be OD, but it avoids modifying the `Address` struct that gets shipped with the SDK.

Reply:
I was thinking it might be something that can be done commonly. Like is this old address same as new address, etc.

Reply:
Yeah it's probably fine. I'm thinking of a dev best practice I saw a while back about how test code can influence production code. [This article](https://8thlight.com/insights/dont-mix-test-code-with-production-code) holds a similar sentiment. But like I said i'm being OD lol. Total take or leave from me.